Automattic/wp-calypso
# Features Grid: adds support for sub features for trail map

Related to #89682

## Proposed Changes

* This PR adds support for display sub-features if they are present.
<img width="283" alt="image" src="https://github.com/Automattic/wp-calypso/assets/5436027/eeb8f147-8652-449f-8e1c-055554c23206">


## Testing Instructions



* Run `yarn workspace @automattic/plans-grid-next storybook`.
* Check the "Trail Map Copy" and "Trail Map Copy & Structure" stories and confirm that sub features displayed for the `25+ pre-installed ecommerce plugins ($1,500/year value)` feature matches the design (Uzd0AYiDAV4eFq1V35T9Ef-fi-800_19072)

## Pre-merge Checklist
